### PR TITLE
Extentions/GitHub/meshmap :  Add GitHub Action for MeshMap Snapshot 

### LINKED ISSUE #7880: [UI] Extensions: Add GitHub Action for MeshMap Snapshot
### Current Behavior
An early release of a Meshery extension as a GitHub Action, [MeshMap Snapshot](https://github.com/marketplace/actions/meshmap-snapshot), is publicly available and ready to be listed in the Meshery UI Extensions page.

### Desired Behavior
Every Meshery GitHub Action should be represented as a card on the extensions page. Each card should have an enable that will invoke the GitHub deployment operation.

### Implementation
1. Create Card. Add description: "Connect MeshMap to your GitHub repo and see changes pull request-to-pull request.Get snapshots of your infrastructure directly in your PRs."
1. Create "Enable" button. "Enable" button should link to https://cloud.meshery.io/connect/github/new/

### Acceptance Tests
1. Ensure "Enable" button opens GitHub Action connection wizard on click.

### Mockups
Screenshot 2023-06-16 at 11 59 40 AM

Link to design - https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI?type=design&node-id=6671-28631&t=ktzjesEfBrsTeEOT-4

---

#### Contributor [Guides](https://docs.meshery.io/project/contributing) and [Handbook](https://layer5.io/community/handbook)

- 🎨 Wireframes and designs for Meshery UI in [Figma](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI)
- 🖥 [Contributing to Meshery UI](https://docs.meshery.io/project/contributing/contributing-ui)
- 🙋🏾🙋🏼 Questions: [Discussion Forum](https://discuss.layer5.io) and [Community Slack](http://slack.layer5.io)

### PR DESCRIPTION
**Pull request by aabidsofi19**
**Notes for Reviewers**

This PR fixes #7880

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by Chadha93** [comment]
@aabidsofi19 Thanks for the PR. Could you please add a few screenshots of how it looks? :ok_hand:

**Comment by aabidsofi19** [comment]
Sure

File: ui/pages/extensions.js [hunk 1]
```diff
@@ -94,7 +94,57 @@ const MeshMapSignUpcard = ({ classes, hasAccessToMeshMap=false }) => {
...
+
+const MeshMapSnapShotCard = ({ classes,githubActionEnabled=false }) => {
+
+  const handleEnable = (e) => {
+    window.open("https://cloud.meshery.io/connect/github/new/", "_blank")
+    e.stopPropagation();
+  };
+
+  return (
+    <Grid item {...INITIAL_GRID_SIZE}>
```
Comment (leecalcote) [inline ui/pages/extensions.js:120]: Is `INITIAL_GRID_SIZE` defined somewhere?

File: ui/pages/extensions.js [hunk 2]
```diff
@@ -94,7 +94,57 @@ const MeshMapSignUpcard = ({ classes, hasAccessToMeshMap=false }) => {
...
 
+const LogoStyles = () => ({
+
+  img : {
+    paddingRight : "1rem",
+    height : "auto",
+    width : "auto",
+    maxWidth : "220px",
+    maxHeight : "150px",
+  },
```
Comment (leecalcote) [inline ui/pages/extensions.js:105]: Good of you to consider the responsiveness of these images.

File: ui/public/static/img/meshmap-snapshot-logo.svg
Comment (leecalcote) [inline ui/public/static/img/meshmap-snapshot-logo.svg:1]: Do not wrap PNGs into an SVG. Please either keep the two SVGs separate or combine them into a single file... as SVGs.

**Review by leecalcote — CHANGES_REQUESTED** [review]

File: ui/pages/extensions.js (hunk 1 above)
Comment (aabidsofi19) [reply ui/pages/extensions.js:120]: Yeah line 64 .

File: ui/public/static/img/meshmap-snapshot-logo.svg
Comment (aabidsofi19) [reply ui/public/static/img/meshmap-snapshot-logo.svg:1]: Oh . I  combined them in figma myself 😁 , as i didn't have copy access . Now Yash has given me the excess .

Comment (leecalcote) [reply ui/public/static/img/meshmap-snapshot-logo.svg:1]: ok, good deal.

**Comment by vishalvivekm** [comment]
@aabidsofi19 
Let's discuss this on Meshery Dev call. Please add this as an agenda item in the meeting minutes if you would. :)

**Review by Abhishek-kumar09 — APPROVED** [review]

**Comment by Chadha93** [comment]
Yay @aabidsofi19 :balloon:

**Pull request merged**